File: homework_6/Presenter.java

//класс запуска калькулятора
public class Presenter {
    public static void main(String[] args){
        view.ParseLine v = new view.ParseLine();
        Model calculator = new Model(v.getA(), v.getOperation(), v.getB());
        view.Result.result(calculator.getInstance().getResult());
        LoggerCalc.myLog(view.ResultStrLog.myLog(v.getA(), v.getOperation(), v.getB(), calculator.getInstance().getResult()));
    }
}

File: homework_7/presenter.py
import view.parse_line
import view.result
import model

"""запуск калькулятора"""
list_ex = []
list_ex = view.parse_line.parse_line()

res = model.get_instance(view.parse_line.get_operation(list_ex), 
                            view.parse_line.get_a(list_ex), view.parse_line.get_b(list_ex))

print(f"{view.parse_line.get_a(list_ex)} {view.parse_line.get_operation(list_ex)} "
        + f"{view.parse_line.get_b(list_ex)} = {view.result.result(res)}")

